### DISCUSSION TITLE
Why 7zip extracted failed?

### DISCUSSION THREAD

**Discussion by pigrich** [microsoft/vcpkg#42741] [Q&A]
![image](https://github.com/user-attachments/assets/5859a15c-ccad-4ee6-870e-ffb8b9b6ec46)
Now I can't install any packages because of failed extraction of 7zip, which really annoys me.

**Comment by pigrich** [comment]
Well, it seems that the directory `downloads/tools/` in my local repo has a problem. I cloned the repo into the other location, and installed a package and then 7zip was extracted successfully! So I got two folders, `7zip-24.09-windows` and `7zr-24.09-windows` in the directory `downloads/tools/`. I copied them into the original repo and now the repo can install packages properly.

**Comment by 1093791954** [comment]
`\vcpkg\downloads\7z2409.7z` extracting `\vcpkg\downloads\tools\7zip-24.09-windows`
